catalog/ui/search-ui/standard/pom.xml

```diff
@@ -214,9 +214,16 @@
                 <groupId>org.asciidoctor</groupId>
                 <artifactId>asciidoctor-maven-plugin</artifactId>
                 <version>1.5.2</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.asciidoctor</groupId>
+                        <artifactId>asciidoctorj-pdf</artifactId>
+                        <version>1.5.0-alpha.6</version>
```

Comment:
Why do we need an `alpha` version of this dependency? Seems like there is a 1.5.4 released: https://github.com/asciidoctor/asciidoctorj/releases/tag/v1.5.4 or am I missing something?